.stylelintrc

```diff
@@ -44,7 +44,8 @@
     "plugin/no-unsupported-browser-features": [
       true,
       {
-        "ignorePartialSupport": true
+        "ignorePartialSupport": true,
+        "ignore": ["css-nesting", "css-when-else", "flexbox-gap"]
```

Comment:
The new plugin version learned about "css-nesting" and "css-when-else", but now mistakes our SASS code for these features. 😅